packages/database/src/migrations/20210920050147-NetEnrolmentRatesMapOverlayLaos-modifies-data.js

```diff
@@ -0,0 +1,256 @@
+'use strict';
+
+import { insertObject, generateId } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const PERMISSION_GROUP = 'LESMIS Public';
+
+const MAP_OVERLAY_GROUP = {
+  id: generateId(),
+  name: 'Net Enrolment Rate',
+  code: 'Laos_Schools_Net_Enrolment_Rate_Group',
+};
+
+const DISTRICT_OVERLAY_GROUP = {
+  id: generateId(),
+  name: 'District Level',
+  code: 'Laos_Schools_Net_Enrolment_Rate_District_Level_Group',
+};
+
+const PROVINCE_OVERLAY_GROUP = {
+  id: generateId(),
+  name: 'Province Level',
+  code: 'Laos_Schools_Net_Enrolment_Rate_Province_Level_Group',
+};
+
+const MAP_OVERLAYS = [
+  {
+    dataElement: 'ner_district_pe_t',
+    name: 'NER Primary',
+    reportCode: 'LESMIS_ner_primary_district',
+    mapOverlayGroupId: DISTRICT_OVERLAY_GROUP.id,
+    sortOrder: 0,
+  },
+  {
+    dataElement: 'ner_district_lse_t',
+    name: 'NER Lower Secondary',
+    reportCode: 'LESMIS_ner_lower_secondary_district',
+    mapOverlayGroupId: DISTRICT_OVERLAY_GROUP.id,
+    sortOrder: 1,
+  },
+  {
+    dataElement: 'ner_district_use_t',
+    name: 'NER Upper Secondary',
+    reportCode: 'LESMIS_ner_upper_secondary_district',
+    mapOverlayGroupId: DISTRICT_OVERLAY_GROUP.id,
+    sortOrder: 2,
+  },
+  {
+    dataElement: 'ner_province_pe_t',
+    name: 'NER Primary',
+    reportCode: 'LESMIS_ner_primary_province',
+    mapOverlayGroupId: PROVINCE_OVERLAY_GROUP.id,
+    sortOrder: 0,
+  },
+  {
+    dataElement: 'ner_province_lse_t',
+    name: 'NER Lower Secondary',
+    reportCode: 'LESMIS_ner_lower_secondary_province',
+    mapOverlayGroupId: PROVINCE_OVERLAY_GROUP.id,
+    sortOrder: 1,
+  },
+  {
+    dataElement: 'ner_province_use_t',
+    name: 'NER Upper Secondary',
+    reportCode: 'LESMIS_ner_upuer_secondary_province',
+    mapOverlayGroupId: PROVINCE_OVERLAY_GROUP.id,
+    sortOrder: 2,
+  },
+];
+
+const getMapOverlayGroupId = async function (db, code) {
```

Comment:
For future reference, there is a utils in `src/utilities` called `codeToId` that could be used in place of this function to clean up the files a bit.